### DISCUSSION TITLE
Ability to rename profiles

### DISCUSSION THREAD

**Discussion by pablomalo** [floccusaddon/floccus#1482] [Ideas]
First I wanted to let you know that I really appreciate this app.

I'm taking advantage of the ability to use multiple profiles. All my profiles sync to the same server (albeit to different folders).

In the Android app, it seems the profiles are automatically named after the server.

This means all my profiles have the same name, which makes it pretty much impossible to select the right one when I'm sharing a link from the browser.

It would help to let the user assign a custom name to each profile, in my opinion.

What do you think?

**Comment by marcelklehr** [comment]
Hi @pablomalo 

Thank you for the feedback. Indeed, this is being tracked over here: https://github.com/floccusaddon/floccus/issues/1294
